src/marqo/tensor_search/models/index_settings.py

```diff
@@ -161,7 +163,10 @@ def to_marqo_index_request(self, index_name: str) -> MarqoIndexRequest:
             if self.treatUrlsAndPointersAsImages is None:
                 # Default value for treat_urls_and_pointers_as_images is False, but we can't set it in the model
                 # as it is not a valid parameter for structured indexes
-                self.treatUrlsAndPointersAsImages = False
+                if self.treatUrlsAndPointersAsMedia is True:
+                    self.treatUrlsAndPointersAsImages = None #False
+                else:
+                    self.treatUrlsAndPointersAsImages = False
             
```

Comment:
Is this same logic as the one we do in the root validator? Is this duplicate?

Reply:
I deleted the one from above (duplicate) and kept the one in here since it receives the user's settings

Reply:
```
if self.treatUrlsAndPointersAsMedia is True:
      self.treatUrlsAndPointersAsImages = None #False
```
This line is a bit weird. The logic is correct, but we are actually leave it as is when treatUrlsAndPointersAsMedia is True. And we probably should do the conflict check here (treatUrlsAndPointersAsImages == True but treatUrlsAndPointersAsMedia == False) to fail fast.

Reply:
Clarified code